server/db/index.js

```diff
@@ -14,20 +14,21 @@ User.hasOne(Cart);
 Cart.belongsTo(User);
 
 // Many to Many
-const cart_product = db.define('cart_product', {
+const cartProduct = db.define('cartProduct', {
```

Comment:
this could be moved to its own file in /models folder

Reply:
I'd prefer to keep it in this file atleast for now so its clear that it is being created as a through table but if you really want it in its own file I can refactor it later.